functions/log/main.py

```diff
@@ -25,3 +36,33 @@ def hello_world(data, context):
     print('Hello, stdout!')
     logging.warn('Hello, logging handler!')
 # [END functions_log_helloworld]
+
+
+# [START functions_log_retrieve]
+cloud_client = cloud_logging.Client()
+log_name = 'cloudfunctions.googleapis.com%2Fcloud-functions'
+cloud_logger = cloud_client.logger(log_name.format(os.getenv('GCP_PROJECT')))
```

Comment:
Do we mention in the readme or anywhere else that this env variable has to be set for it to work locally?